src/js/tabs/pid_tuning.js

```diff
@@ -1764,26 +1780,56 @@ pid_tuning.initialize = function (callback) {
             context.lineWidth = 2;
             context.strokeStyle = '#ffbb00';
             context.stroke();
+
+            let thrPositionLimit = {};
+            if (throttleLimitType === THROTTLE_LIMIT_TYPES.CLIP) {
+
+                // FIXME Search into the canvas data to find the first non-zero pixel at the limited throttle position
+                // there must be a better way to do this, in a mathematically way
+                thrPositionLimit.y = canvasHeight - (throttleLimitPercent * canvasHeight);
+                const dataCanvas = context.getImageData(0, thrPositionLimit.y, canvasWidth, 1);
+                const RGBA_SIZE = 4;
+                for (let i = dataCanvas.data.length - 1; i >= 4; i -= RGBA_SIZE) {
+                    if (dataCanvas.data[i] !== 0 && dataCanvas.data[i - RGBA_SIZE] === 0) {
+                        thrPositionLimit.x = (i + (canvasWidth / 100) * 2) / RGBA_SIZE;
+                        break;
+                    }
+                }
```

Comment:
Maybe someone can find a better way of doing this. I don't know how to know the "stick" value of a fixed "throttle".
This code looks into the graph for a pixel of the curve, and cuts there, but maybe there is a mathematically way of doig this.
Another option is to call the `calculateThrottlePosition()` with different stick values "searching" the correct throttle result, but I don't like it neither.

Reply:
I think this does what you need:
```
thrPositionLimit = throttleLimitPercent<= mid
                    ? getQuadraticCurvePoint(0, canvasHeight, midxl, midyl, midx, midy, throttleLimitPercent* (1.0 / mid))
                    : getQuadraticCurvePoint(midx, midy, midxr, midyr, canvasWidth, 0, (throttleLimitPercent- mid) * (1.0 / (1.0 - mid)));
```
`thrPositionLimit.x` is x value of limit
`thrPositionLimit.y` is  y value of limit (no search required)